FirebaseCombineSwift/Sources/Auth/Auth+Combine.swift

```diff
@@ -17,6 +17,8 @@
   import Combine
   import FirebaseAuth
 
+  @objc(FIRCombineAuthComponent) private class __CombineAuthComponent: NSObject {}
```

Comment:
Can you add a comment above just mentioning this is exclusively used to detect the class from ObjC and shouldn't be used directly?

Reply:
LMK if this is what you had in mind:

```
// Make this class discoverable from Objective-C. Don't instantiate directly.
```